app/controllers/registrations_controller.rb

```diff
@@ -41,18 +41,12 @@ def new
 
   # POST /resource
   def create
-    #logger.debug "#{sign_up_params}"
     if !sign_up_params[:accept_terms] then
       redirect_to after_sign_up_error_path_for(resource), alert: _('You must accept the terms and conditions to register.')
     else
-      existing_user = User.find_by_email(sign_up_params[:email])
-      if !existing_user.nil? # If email exists
-        if (existing_user.password == "" || existing_user.password.nil?) && existing_user.confirmed_at.nil? # If user has not accepted invitation yet
```

Comment:
Why remove this? It was added to handle situations where a invitation email was sent but the user tries to create an account outside the invitation process. See #322. Devise creates the user record in the DB when it sends out the invitation so the user cannot create an account because the email is already known.

Reply:
The logic introduced covers more sensibly the edge case for #322. Note before it was just a super, not executing our override lines for when a user does not exist.

Reply:
ok. will test it out once deployed to make sure #322 hasn't been reintroduced